qsim.cpp

```diff
@@ -270,6 +270,12 @@ const char** get_qemu_args(const char* kernel, int ram_size, int n_cpus, const s
     NULL
   };
 
+  /*
+  "-s",
+  "-S",
+  "-gdb", "tcp::1234",
+  */
+
```

Comment:
Yes, please. I understand that we want some code to debug. For such cases, it makes sense to use a macro like DEBUG_QSIM and place all the debug code within it.